secret/secret_test.go

```diff
@@ -619,6 +619,89 @@ func TestUpdateVariable(t *testing.T) {
 		configVar := model.ConfigurationVariable{Name: "IN_UPDATES"}
 		result := updateVariable(&secrets, &update, &configVar)
 		assert.False(result)
-		assert.Equal(string(secrets.Data["in-updates"]), "orig")
+		assert.Equal("orig", string(secrets.Data["in-updates"]))
+	})
+}
+
+func TestMigrateRenamedVariable(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
Is this correct? Doesn't the `assert` object need to be created from the `t` of the individual sub-tests?